### ISSUE TITLE
Error in Listview with previously selected rows and changing to next page (pagination) - Bulk Action disabled and wrong "Selected" count

### ISSUE THREAD

**Issue by lenz1979** [salesagility/SuiteCRM#4297]
#### Issue

In Accounts-Listview i select e.a. the first to rows (for using with Bulk-Action).
Then change to page number 2 using paginagion button (Next).
On page 2 select some more Rows.
Change back to page number 1 using pagination button (prev).

the 2 previously selected rows are still selected (correct), but its not possible to trtigger any "Bulk Action" because the button is disabled.

if i deselect one of the selected rows it shows "Selected: -1".

#### Expected Behavior

the number of selected items/rows should be correct even when changing to next page.
after changing to nexxt page (using pagination) the "Bulk ActioN" should not be disabled if previously selected any item/rot.

#### Actual Behavior

see description on top. "Bulk Action" is disabled on second page and "Selected" shows wrong counter.

#### Steps to Reproduce

1. select rows in accounts list
2.go to page number 2
3.select some more items and go back using paginiation

#### Your Environment

* SuiteCRM Version used:
7.9.4
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
firefox/chrome
* Environment name and version (e.g. MySQL, PHP 7):
php 5.6
* Operating System and version (e.g Ubuntu 16.04):
debian 7

**Comment by lenz1979** [comment]
may be fixed by modifying include/ListView/ListViewDisplay.php (Funktion buildSelectedObjectsSpan) by adding:


`if(isset($_POST["mass"]))
  $total = count($_POST["mass"]);`

**Comment by TeuZ63** [comment]
1 year and 6 months after you, i had the same problem !
I found a solution :

In the file "jssource\src_files\include\javascript\sugar_3.js", the function "sugarListView.get_num_selected" return a strange value. I changed the function :

Before :

```
sugarListView.get_num_selected = function () {
  var selectCount = $("input[name='selectCount[]']:first");
  if (selectCount.length > 0) 
         return parseInt(selectCount.val().replace("+", ""));
  return 0;
}
```
Now : 
```
sugarListView.get_num_selected = function () {
   var selectCount = sugarListView.get_checks_count();
   if (selectCount > 0)
         return selectCount;
   return 0;
}
```
It's works, but the code the code has been changed in the "Core" ...
I'm not familiar with GitHub and project contribution and I don't know how the community can validate this change ?

**Comment by Dillon-Brown** [comment]
Hi @TeuZ63, thanks for the fix, seemed to work ok for me during my initial testing. If you were interested in contributing this fix so it can be merged into core would you be able to make a pull request?

The documentation to do so is here: [Bug Fixes](https://docs.suitecrm.com/community/contributing-code/bugs/). Essentially you would [clone down this repo](https://docs.suitecrm.com/community/contributing-code/forking/), checkout the hotfix-7.8.x branch, create a new bugfix branch with this issue (4297) in the name, make your changes, commit and push them up to your fork, and create a pull request to the main SalesAgility repo before repeating the process for the hotfix-7.10.x branch.

If you encounter any difficulty just let me know and we can help you out, otherwise, if you'd prefer I can make the pull request for you. Thanks!

**Comment by TeuZz** [comment]
Thanks @Dillon-Brown. I will try to create a pull request myself.
I come back here if any problem occurs.

**Comment by TeuZz** [comment]
@Dillon-Brown , sorry for disturbance.
Can you check my two "pull request" are fine ?
It's my first GitHub contribution, and I want do well.

**Comment by Dillon-Brown** [comment]
Hi @TeuZz, look perfect. Would you also be able to sign the "ClaHub" check in your PR as well? you should be able to see at the bottom with an option to click on "Details" that will take you through. Thanks!

**Comment by pgorod** [comment]
@TeuZz it seems there is a recession with this fix, please see Issue #7304 	

Can you figure out what went wrong there? Thanks

**Comment by lazka** [comment]
I can still reproduce this, fixed with #7538

**Comment by rajadsms** [comment]
I am still getting this issue, I am using React-admin 3.14.0
Can anyone help me with this?
cc: @TeuZz @lazka @Dillon-Brown
